doc/classes/BitMap.xml

```diff
@@ -9,6 +9,45 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="bitwise_and">
+			<return type="BitMap" />
+			<param index="0" name="other" type="BitMap" />
+			<param index="1" name="size_mode" type="int" enum="BitMap.BitwiseSizeMode" default="3" />
+			<description>
+				Creates a new bitmap from the boolean AND operation with [param other].
+				[param size_mode] defines the size of the new bitmap based on this bitmap size and [param other] size.  If this bitmap has the same size as [param other], the new bitmap will always be that size.
+			</description>
+		</method>
+		<method name="bitwise_not">
+			<return type="BitMap" />
+			<description>
+				Creates a new bitmap from the boolean NOT operation (i.e. every bit has been flipped from [code]true[/code] to [code]false[/code] and [code]false[/code] to [code]true[/code]).
+			</description>
+		</method>
+		<method name="bitwise_or">
+			<return type="BitMap" />
+			<param index="0" name="other" type="BitMap" />
+			<param index="1" name="size_mode" type="int" enum="BitMap.BitwiseSizeMode" default="3" />
+			<description>
+				Creates a new bitmap from the boolean OR operation with [param other].
+				[param size_mode] defines the size of the new bitmap based on this bitmap size and [param other] size.  If this bitmap has the same size as [param other], the new bitmap will always be that size.
+			</description>
+		</method>
+		<method name="bitwise_xor">
+			<return type="BitMap" />
+			<param index="0" name="other" type="BitMap" />
+			<param index="1" name="size_mode" type="int" enum="BitMap.BitwiseSizeMode" default="3" />
+			<description>
+				Creates a new bitmap from the boolean XOR operation with [param other].
+				[param size_mode] defines the size of the new bitmap based on this bitmap size and [param other] size.  If this bitmap has the same size as [param other], the new bitmap will always be that size.
+			</description>
+		</method>
+		<method name="convert_to_image" qualifiers="const">
```

Comment:
convert to image docs are included twice for some reason

Reply:
Ah, that's my bad.  I was having trouble with the documentation and I ended up copying on of the existing methods to fix it.  It is fixed now.